Gemfile.lock

```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     enumerate_it (1.2.8)
-      activesupport (>= 3.0.0)
+      activesupport (= 3.0.5)
```

Comment:
I'll let this to another pull request, since the interface significantly changed :dancers: